capstone/src/main/java/com/step902020/capstone/IndividualController.java

```diff
@@ -20,14 +20,28 @@
   @Autowired
   private IndividualRepository individualRepository;
 
-  /** Find an individual's profile information by email */
+  @Autowired
+  private EventRepository eventRepository;
+
+  /** Find an individual's profile information by email
```

Comment:
We usually start the comment on the next line after `/**`, but it doesn't really matter for javadoc.
Btw, you can test the javadoc by running `mvn javadoc:javadoc` goal.